chart/compass/values.yaml

```diff
@@ -298,6 +298,7 @@ global:
     runtimeTypeLabelKey: "runtimeType"
     applicationTypeLabelKey: "applicationType"
     kymaRuntimeTypeLabelValue: "kyma"
+    kymaApplicationNamespaceValue: "sap.kyma"
```

Comment:
I was considering it but decided against this approach because to me it looked that it's not worth it maintaining an override for such a 'narrow scoped'/'single use case' variable.

In addition, currently we don't have an override for the similar `kymaRuntimeTypeLabelValue` variable and decided to keep it consistent in that aspect too.

What's your opinion? I am open for a discussion.